src/languageSupport/languages/influxql/connection.ts

```diff
@@ -21,6 +22,26 @@ import {compositionEnded} from 'src/shared/copy/notifications'
 import {groupedTagValues} from 'src/languageSupport/languages/agnostic/utils'
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 
+/*
```

Comment:
did you mean to leave this comment here?

Reply:
Yeah, just to make it easier to understand what the function does

Reply:
fwiw i find writing a utils test for a helper function a good way to demonstrate what a function does / prevents people misinterpreting and breaking it

Reply:
Good call! I'll move that into a util with a unit test